src/controllers/users.js

```diff
@@ -1,58 +1,77 @@
-import { Router } from 'express';
 import HttpStatus from 'http-status-codes';
 import * as userService from '../services/userService';
-import { findUser, userValidator } from '../validators/userValidator';
-
-const router = Router();
 
 /**
- * GET /api/users
+ * Get all users
+ * @param {Object} req
+ * @param {Object} res
+ * @param {Function} next
  */
-router.get('/', (req, res, next) => {
-  userService
-    .getAllUsers()
-    .then(data => res.json({ data }))
```

Comment:
I kept using `then` because it looks more cleaner to me.

Reply:
Also, it doesn't need to be `async`.

Reply:
🆗